packages/auth/amplify_auth_cognito_dart/lib/src/model/session/cognito_user_pool_tokens.dart

```diff
@@ -75,18 +75,12 @@ abstract class CognitoUserPoolTokens
   String get username => CognitoIdToken(idToken).username;
 
   /// Validates the tokens against the client state.
-  void validate({
-    String? nonce,
-  }) {
-    // Missing nonce values or nonce mismatches should throw exceptions,
-    // indicating the tokens cannot be used.
-    final idTokenNonce = idToken.nonce;
-    if (nonce == null || idTokenNonce == null) {
-      throw const InvalidStateException('Missing nonce');
-    }
-    if (nonce != idTokenNonce) {
-      throw const InvalidStateException('Nonce values do not match');
-    }
+  void validate() {
```

Comment:
What is the purpose of keeping an empty function? 

Is this still being called somewhere else?

Reply:
It was recommended by Tim so that the context was not lost in the codebase and so that it wasn't reimplemented.

Reply:
It is still being called, yes.